libs/zard/src/lib/shared/components/hover-card/doc/api.md

```
@@ -0,0 +1,25 @@
+# API
+
+### [zHoverCard]
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)